src/main/java/randoop/generation/ForwardGenerator.java

```diff
@@ -151,6 +195,99 @@ public ExecutableSequence step() {
     return eSeq;
   }
 
+  /**
+   * Recompute weights for all methods under test at regular intervals.
+   *
+   * @param interval interval at which to recompute weights
+   */
+  private void processWeightsForMethods(int interval) {
+    // Increment our step counter.
```

Comment:
It's not helpful to give comments that restate the exact operation.  Remove them for brevity.  There are a number of examples like this; another is "clear the method selections map".